dotcom-rendering/src/web/components/Callout/Form.stories.tsx

```diff
@@ -1,60 +1,29 @@
-import { css } from '@emotion/react';
-import {
-	calloutCampaign,
-	calloutCampaignOnlyTwoRadio,
-} from '../../../../fixtures/manual/calloutCampaign';
+import { ArticleDesign, ArticleDisplay, ArticlePillar } from '@guardian/libs';
+import { calloutCampaign } from '../../../../fixtures/manual/calloutCampaign';
 import { Form } from './Form';
 
+const mockFormat = {
```

Comment:
Our new forms theme will change based on the pillar its in, hence we add the `mockFormat` here